content/ri/using-redisinsight/api.md

```diff
@@ -0,0 +1,178 @@
+---
+Title: RedisInsight REST API
+date: 2020-02-02 00:00:00 +0000
+weight: 1000
+categories: ["RI"]
+path: api/
+nextStep:
+    Title: Memory Analysis
+    href: /docs/features/memory-analysis/
+---
+
```

Comment:
@nishantgeorge I'm missing a section here on why we did this.  We wanted this feature to be documented so that people could script / bootstrap a redisinsight with their databases preloaded for automation / testing purposes ?